drivers/image/src/vic_start.c

```diff
@@ -48,33 +62,127 @@ vic_start(void)
     initialize_parameters();
     initialize_filenames();
 
-    // read global settings
-    filep.globalparam = open_file(filenames.global, "r");
-    get_global_param(filep.globalparam);
+    if (mpi_rank == 0) {
+        // read global settings
+        filep.globalparam = open_file(filenames.global, "r");
+        get_global_param(filep.globalparam);
+    }
 
     // Set Log Destination
+    // TBD: Add MPI support to logging: every process should log to its own file
     setup_logging();
```

Comment:
This should be easy enough to do.  I think we just need to pass the `mpi_rank` as an argument to [`setup_logging()`](https://github.com/UW-Hydro/VIC/blob/develop/drivers/shared/src/vic_log.c#L116) to help define the log file names.